src/Lykke.Job.TransactionHandler/Handlers/EthereumCommandHandler.cs

```diff
@@ -79,11 +93,72 @@ public class EthereumCommandHandler
             if (errorMessage != null)
             {
                 await _ethClientEventLogs.WriteEvent(command.ClientId, Event.Error, new { Request = command.TransactionId, Error = errorMessage }.ToJson());
-                await _log.WriteErrorAsync(nameof(EthereumCommandHandler), nameof(Commands.ProcessEthereumCashoutCommand), command.ToJson(), new Exception(errorMessage));
+                await _log.WriteErrorAsync(nameof(EthereumCommandHandler), nameof(ProcessEthereumCashoutCommand), command.ToJson(), new Exception(errorMessage));
```

Comment:
You an use sync version of method, where context could be an object, so you can avoid ```ToJson()``` call